src/style.css

```diff
@@ -0,0 +1,31 @@
+html {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+a {
+  color: green;
+}
+
+h1 {
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
I know that is a bad practice, but in this simple example it works correct. 
but ok, i'll do what you want)